docs/migration/Version_0.5.x_0.7.x.md

```diff
@@ -0,0 +1,287 @@
+BPNs and DIDs) in Catena-X.
+
+In short, it exposes a GZipped list of BPN<>DID tuples, which the connector caches locally and updates periodically.
+This API is called the _Directory API_.
+
+These requests are authenticated by putting the participant's `MembershipCredential` in the `Authorization` header.
+
+### 4.1 Config values
+
+| Helm value                                 | Environment value                 | required | default value | description                               |
+|--------------------------------------------|-----------------------------------|----------|---------------|-------------------------------------------|
+| `controlplane.bdrs.cache_validity_seconds` | `TX_IAM_IATP_BDRS_CACHE_VALIDITY` |          | 600           | time (sec) that the client cache is valid |
+| `controlplane.bdrs.server.url`             | `TX_IAM_IATP_BDRS_SERVER_URL`     | x        |               | base URL of the BDRS server               |
+
+### 4.2 References
+
+- [BDRS Server on GitHub](https://github.com/eclipse-tractusx/bpn-did-resolution-service)
+- [BDRS Server Directory API](https://eclipse-tractusx.github.io/bpn-did-resolution-service/openapi/directory-api/)
+- [BDRS Server Management API](https://eclipse-tractusx.github.io/bpn-did-resolution-service/openapi/management-api/) (
+  not for public consumption)
+
+Further, the BDRS server is available as Helm chart from the
+official [Tractus-X Chart Repo](https://eclipse-tractusx.github.io/charts/dev):
+
+- `bdrs-server`: official distribution using Postgres
+- `bdrs-server-memory`: variant, that uses pure in-mem storage, useful for testing
+
+To see how BDRS could be used in testing/dev etc. check out
+this [test class](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-extensions/bdrs-client/src/test/java/org/eclipse/tractusx/edc/identity/mapper/BdrsClientImplComponentTest.java)
+
+> Note that consuming the Directory API with tools like Postman or Insomnia is possible, but not straight forward and it
+> is out-of-scope for this document.
+
+## 5. EDC Management API
+
+### 5.1 Catalog
+
+- `counterPartyId` is a **required** property for the catalog request in Tractus-X EDC. It's not enforced currently (
+  backward compatibility) but if not provided, the IATP flows won't work and thus the catalog request will fail.
+
+### 5.2 Transfer Process
+
+- `transferType`  is a needed property for the transfer request to be working in tx-edc. `transferTypes` are the
+  supported transfer format in tx-edc. They return as part of the Catalog response as part of distribution format for an
+  asset. In TX-EDC we currently supports `HttpData-PULL`, `AzureStorage-PUSH` and `AmazonS3-PUSH`. The `transferType`
+  is not mandatory (backward compatibility) but if not provided the transfer request will fail.
+
+### 5.3 EDRs
+
+The 0.5.4 EDRs API was removed due the adoption of DPS (data plane signaling) and reborn under a different shape
+in `/v2/edrs`
```

Comment:
Suggest to add:

"Additionally, the option to register a consumer-side HTTP-callback via `receiverHttpEndpoint` in the `POST /transferprocesses` call was removed. Consumer apps should use the `/v2/edrs` API instead."

Reply:
Probably we can add this in the section  `## 2. Changes to EDR handling and APIs` where we mention the removal of the config `backendService.httpProxyTokenReceiverUrl`. We can add the removal of the `receiverHttpEndpoint` in transfer process as well

Reply:
do you mind adding a GitHub Suggestion to the PR? CTRL-G (or CMD-G on macs)...